specification/glossary.md

```diff
@@ -156,7 +156,7 @@ A string logically identifying the subject of evaluation (end-user, service, etc
 
 ### Fractional Evaluation
 
-Assigning flag values randomly according to a configured proportion or percentage (ie: 50/50).
+Pseudorandomly resolve flag values using a context property, such as a targeting key, based on a configured proportion or percentage (ie: 50/50).
```

Comment:
Should we say that the value should be always the same for the a user?

Reply:
I think that might be too detailed, but that's just my opinion. I like to keep the subject (user/client) vague generally, because I think there are valid non-user driven use cases... clients are not necessarily end-users.

I think this gets the idea across that some property (possibly a user identifier) will allow the split to be deterministic, so that it will be the same for that user over time. We could go into more detail in the doc, perhaps.

Reply:
Yes the glossary is probably not the best place to put this, we may consider to put that in the spec.